benchexec/tablegenerator/template.html

```diff
@@ -348,6 +354,7 @@
 
 $(document).ready(function() {
   $('body').keydown(hideContentPane);
+  $("#dataTable").floatThead({scrollingTop:'1.6em', useAbsolutePositioning:false, zIndex:3});
```

Comment:
`1.6em` is indeed a magic value, yet specifying the dimensions in `em` is not that evil, since it scales with the text size (ctrl+ or ctrl-).
There are two options for floatthead: either absolute positioning, or positioning inside the container (with overflow:auto). With the absolute positioning we need to know the offset in advance; I guess one way is to dynamically query for the size of the top-bar with javascript.